source/backend/dal/Repositories/ProductRepository.cs

```diff
@@ -28,11 +28,12 @@ public ProductRepository(PimsContext dbContext, ClaimsPrincipal user, ILogger<Pr
         /// </summary>
         /// <param name="projectId"></param>
         /// <returns></returns>
-        public IList<PimsProduct> GetByProject(int projectId)
+        public IList<PimsProduct> GetByProject(long projectId)
         {
             return this.Context.PimsProducts.AsNoTracking()
-                .Where(o => o.ParentProjectId == projectId)
-                .OrderBy(a => a.Code)
+                .Where(p => p.ParentProjectId == projectId)
+                .Include(p => p.PimsAcquisitionFiles)
```

Comment:
ok, yes, this is fine by me, looks like we aren't digging into the acq file itself which is in line with our best practices on nested entity loading.